rotkehlchen/inquirer.py

```diff
@@ -556,6 +571,24 @@ def find_price(
         )
         return price
 
+    @staticmethod
+    def find_price_and_oracle_and_check_main_currency(
```

Comment:
There is no reason to create a wrapper function that just calls one single function with different parameters. Please don't do that

Reply:
This is not due to different call arguments, but due to different return types. It returns an extra boolean.

Same approach was previously used for `find_price` and `find_price_with_oracles`, so I just followed this pattern.

Also creating this extra function allows to avoid modifying every single place where `price_with_oracles` is called (since there is no change in returned data).

Reply:
Even though please don't do it. You are using it in ony a single place. Never do that. It makes no sense.